seedemu/core/InternetExchange.py

```diff
@@ -19,13 +19,14 @@ class InternetExchange(Printable, Configurable):
     __rs: Node
     __name: str
 
-    def __init__(self, id: int, prefix: str = "auto", aac: AddressAssignmentConstraint = None):
+    def __init__(self, id: int, prefix: str = "auto", aac: AddressAssignmentConstraint = None, create_rs = True):
         """!
         @brief InternetExchange constructor.
 
         @param id ID (ASN) for the IX.
         @param prefix (optional) prefix to use as peering LAN.
         @param aac (option) AddressAssignmentConstraint to use.
+        @param create_rs (optional) create route server node for the IX or not. (only relevant for BGP)
```

Comment:
"only releveant for BGP" makes is sound like the parameter is ignored in SCION networks. Is that the intention?